src/main/java/lotto/InputView.java

```diff
@@ -0,0 +1,17 @@
+package lotto;
+
+import java.util.Scanner;
+
+public class InputView {
+	private static Scanner scanner = new Scanner(System.in);
```

Comment:
final을 깜빡 하신 것 같아요  :)
```suggestion
	private static final Scanner scanner = new Scanner(System.in);
```